docs/file-upload-with-graphql.md

```diff
@@ -0,0 +1,200 @@
+# File upload with graphql
+
+## Overview
+
+This document discuss the issue of uploading files with graphql. \
+As vanilla graphql does not support uploading files, there are several workarounds
+
+### File uploading with gqlgen 0.9.0 library
+<details>
+<summary>Example of schema.graphql</summary>
+<p>
+
+```graphql
+#"The `Upload` scalar type represents a multipart file upload."
+scalar Upload
```

Comment:
you can mention that Scalar is already implemented by github.com/99designs/gqlgen/graphql/upload.go